lib/gitter-adapter.js

```diff
@@ -109,6 +109,13 @@ Adapter.prototype.setup = function(token) {
   this.loggableToken = obfuscateToken(token);
   debug('setup', { token: this.loggableToken });
 
+  if (token === "dbff95de400bdb378e908dc6ad7b51b029bb493d") {
+      var mask = ':gitter!gitter@irc.gitter.im';
```

Comment:
Yeah worth doing a log.

The username/nickname is in `this.client.nick` if you want to log it out.